common/shlibs

```diff
@@ -3980,7 +3980,7 @@ libopenaptx.so.0 libopenaptx-0.2.0_1
 libsimavr.so.1 simavr-1.6_2
 libsimavrparts.so.1 simavr-1.6_2
 libsword-1.8.1.so libsword-1.8.1_6
-libpari-gmp.so.7 pari-2.13.0_1
+libpari-gmp-tls.so.7 pari-2.13.2_1
```

Comment:
Don't you kind of need both versions here, since it can be built without `pthreads`?

Reply:
Alternatively, completely remove the build option and force pthreads always :p

Reply:
You are right, I will fix that.